src/hint_processor/builtin_hint_processor/secp/bigint_utils.rs

```diff
@@ -93,7 +93,7 @@ impl BigInt3<'_> {
         limbs
             .into_iter()
             .enumerate()
-            .map(|(idx, value)| value.to_bigint().shl(idx * 86))
+            .map(|(idx, value)| value.to_signed_felt().shl(idx * 86))
```

Comment:
This one was probably intended to only be a sign change. The limbs are supposed to be 86 bits, so they are always positive in the field. So I would leave it as `to_bigint`.

Reply:
Without changing this, `tests::cairo_run_test::secp_ec` was failing.